migrations/migrations/006-form-users.js

```diff
@@ -0,0 +1,12 @@
+const createFormUsersTable = `
+  CREATE TABLE form_users (
+    id SERIAL PRIMARY KEY,
+    template_id INT NOT NULL REFERENCES templates (id),
```

Comment:
There needs to be a combined unique constraint on template_id and email to ensure emails don't get added more than once a particular form